nc-news/src/components/CommentCard.jsx

```diff
@@ -24,26 +24,46 @@ function CommentCard({ article_id }) {
   function timeStamp(comment) {
     return (
       comment.created_at.slice(0, 10) + " " + comment.created_at.slice(11, 16)
-    );
+    )}
+
+  function handleCommentDelete(comment_id) {
+    setDeleteLoading(true);
+    axios
+      .delete(`https://be-nc-news-zmuo.onrender.com/api/comments/${comment_id}`)
+      .then(() => window.location.reload())
```

Comment:
I understand why this is here so that the page reloads to show the new comment. However. I would strongly recommend not reloading the whole page. You could use some kind of state or optimistically render the comment being removed.